account_check_printing_report_base/README.rst

```diff
@@ -0,0 +1,109 @@
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-payment/tree/12.0/account_check_printing_report_base
+    :alt: OCA/account-payment
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-payment-12-0/account-payment-12-0-account_check_printing_report_base
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/96/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides the basic framework for check printing, and a sample
+layout.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+In order to install this module you must first install also the module
+'report_paper_wkhtmltopdf_params', available in
+https://github.com/OCA/server-tools
+
+Configuration
+=============
+
+Go to 'Settings / Users / Companies' and assign the desired check format.
+This module proposes a basic layout, but other modules such as
+"account_check_printing_report_dlt103" provide formats adjusted to known
+check formats such as DLT103.
+
+Usage
+=====
+
+* Go to 'Invoicing / Purchases / Payments'. Select one of the payments with
+  type 'Check' and print the check.
+* For automatic check printing when validating payment, mark the field in
+  the journal associated.
+
+Known issues / Roadmap
+======================
+
+* When print check automatically in the payment validation process, the wizard
+  remains opened.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/account-payment/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/account-payment/issues/new?body=module:%20account_check_printing_report_base%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Eficent
+* Serpent Consulting Services Pvt. Ltd.
+* Ursa Information Systems
+
+Contributors
+~~~~~~~~~~~~
+
+* Jordi Ballester Alomar <jordi.ballester@eficent.com>
+* Lois Rilo Antelo <lois.rilo@eficent.com>
+* Sandip Mangukiya <smangukiya@ursainfosystems.com>
+* Luis M. Ontalba <luis.martinez@tecnativa.com>
+* Manuel Marquez <mmarquez@iterativo.do>
```

Comment:
Seems like you didn't regenerate the README, and the changes to the Contributors file didn't apply.